ValeStyles/Couchbase/An.yml

```diff
@@ -14,7 +13,7 @@ exceptions:
   - '[Aa] [Uu]ser'
   - '[Aa] [Uu]sername'
   - '[Aa] UI'
-  - '[Aa] <<[BbCcDdFfGgHhJjKkLlMmNnPpQqRrSsTtVvWwXxYyZz].+>>'
+  - '\b[Aa] [^aeiouAEIOU\s][a-zA-Z]*'
```

Comment:
this is for non-vowels? it wouldn't be matched by the token above, so is an odd exception?

Reply:
True. I think it was just a holdout because I was running into issues with the original regex. Let me correct that...